### PR TITLE
Script for cross compiling for multiple platforms

### LINKED ISSUE #38: Build scripts for henge
We need to have build scripts for henge.

Requirements : 
1. Should build henge binaries for all supported $GOOS
2. Should work on developer machines
3. Should work on travis for all supported platforms (if possible)

### PR DESCRIPTION
**Pull request by kadel**
This adds script `./hack/build-cross.sh` that makes easy to build Henge binary for multiple platforms.

fixes #38 

and related to #36

### CODE REVIEW

File: hack/build-cross.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PLATFORMS=${1:-"darwin linux windows"}
+ARCHS=${2:-"amd64"}
+
+echo "Staring builds"
+
+for platform in $PLATFORMS; do
+  for arch in $ARCHS; do
+    echo " Building for ${platform}/${arch}"
+    OUTPUT_FILE=${BUILD_DIR}/${platform}/${arch}/henge
+    OOS=$platform GARCH=$arch go build -o ${OUTPUT_FILE}
```
Comment (pradeepto) [inline hack/build-cross.sh]: @kadel  Shouldn't this be GOOS?

Comment (kadel) [inline hack/build-cross.sh]: oh, yes

**Comment by kadel** [comment]
I still need to test darwin an windows builds, but i don't have mac/win machine right now :-(

File: hack/build-cross.sh (hunk 1 above)
Comment (kadel) [inline hack/build-cross.sh]: fixed

**Comment by kadel** [comment]
I got my hands on OSX and Win and tested it. I can confirm that it now works everywhere ;-)

**Comment by surajssd** [comment]
@kadel nice!

**Comment by pradeepto** [comment]
:+1:  Awesome @kadel

**Comment by cdrage** [comment]
This would be a good case to add a Makefile to henge so it's easier to exec the builds / scripts :)

**Comment by cdrage** [comment]
Otherwise, LGTM!

**Comment by kadel** [comment]
Good point about makefile @cdrage. I have created new issue for it #54

**Pull request merged**